oauth2client/contrib/gce.py

```diff
@@ -134,8 +133,8 @@ def _refresh(self, http_request):
             self._retrieve_info(http_request)
             self.access_token, self.token_expiry = _metadata.get_token(
                 http_request, service_account=self.service_account_email)
-        except httplib2.HttpLib2Error as e:
-            raise client.HttpAccessTokenRefreshError(str(e))
+        except transport.transport_error() as err:
```

Comment:
Yes, I'm not sure I see what needs to be changed here. The idea is if the transport fails to talk to the metadata server, that's equivalent to an AccessTokenRefreshError.

I'm open to ideas, what would you recommend instead?